docs/source/trainer/checkpointing.rst

```diff
@@ -583,7 +583,7 @@ to the environment variables ``GCS_KEY`` and ``GCS_SECRET`` respectively or the
 Saving and Loading Sharded Checkpoints
 --------------------------------------
 Using `FSDP <https://pytorch.org/docs/stable/fsdp.html>`__, you can save and load sharded checkpoints with Composer.
-See `these docs </notes/distributed_training.html#saving-and-loading-sharded-checkpoints-with-fsdp>`_ for more info and some examples.
+See `these docs <projects/composer/en/stable/notes/distributed_training.html#saving-and-loading-sharded-checkpoints-with-fsdp>`_ for more info and some examples.
```

Comment:
```suggestion
See :doc:`these docs </notes/distributed_training.html#saving-and-loading-sharded-checkpoints-with-fsdp>`_ for more info and some examples.
```
I think this is the right fix, but also you can check locally with
```
cd docs
make html
make host
```

Reply:
works for me locally

Reply:
we don't want to include `projects/composer/en/stable` because then it would be the wrong prefix for other versions of the docs like `latest`. You can also view the docs built from this pr at https://mosaicml-docsite--2278.com.readthedocs.build/projects/composer/en/2278/trainer/checkpointing.html, which results in the link https://mosaicml-docsite--2278.com.readthedocs.build/projects/composer/en/2278/trainer/projects/composer/en/stable/notes/distributed_training.html#saving-and-loading-sharded-checkpoints-with-fsdp, which 404s for me.

Reply:
Ok, I think this should actually fix it.